registrations/forms.py

```diff
@@ -0,0 +1,31 @@
+from django import forms
+
+from registrations.models import RegistrationPriceGroup
+
+
+class RegistrationPriceGroupAdminForm(forms.ModelForm):
+    price = forms.DecimalField(
+        required=False, max_digits=19, decimal_places=2, min_value=0
+    )
+    price_without_vat = forms.DecimalField(
+        required=False, disabled=True, max_digits=19, decimal_places=2, min_value=0
+    )
+    vat = forms.DecimalField(
+        required=False, disabled=True, max_digits=19, decimal_places=2, min_value=0
+    )
```

Comment:
The form is initialized with 4 decimals and limiting decimal amount to 2 causes validation problems. Basically user is not able to edit price groups because `price_without_vat` and `vat` fields are disabled

<img width="1361" alt="Screenshot 2024-01-04 at 14 43 15" src="https://github.com/City-of-Helsinki/linkedevents/assets/24706814/50efe4ec-dc5b-428d-8e4f-33ead8346f46">

Reply:
Should be fixed now by allowing only two decimal places for all price fields in the database (as the Euro currency uses only two decimal places)